### DISCUSSION TITLE
Production build not working

### DISCUSSION THREAD

**Discussion by im-ghost** [vercel/next.js#46709] [Help] [answered]
### Summary

I have a next js project which works perfectly fine on localhost but doesn't work in production. 

### Additional information
[GitHub url](https://github.com/im-ghost/apish)
[production buil](https://63fe55451ecc1c26b51af4af--majestic-tapioca-dcc79c.netlify.app/)

**Comment by ciruz** [comment]
Hey a little more context would be useful.

Maybe there is something wrong with the environment variables / env files, your build settings, deployement process. I would check for any differences between local and production system, could also be a different node.js version.

**Reply by im-ghost** [reply]
I thought I added the GitHub url and the deployment url
Already added

**Reply by ciruz** [reply]
the repository must be private, can't access it.

**Reply by im-ghost** [reply]
Forgive me 
Already made it public

**Comment by im-ghost** [comment, marked as answer]
Fixed
